Gemfile

```diff
@@ -36,10 +37,17 @@ gem 'tzinfo-data', platforms: [:mingw, :mswin, :x64_mingw, :jruby]
 group :development, :test do
   # Call 'byebug' anywhere in the code to stop execution and get a debugger console
   gem 'byebug', platforms: [:mri, :mingw, :x64_mingw]
+  gem 'factory_bot_rails', '~> 5.0', '>= 5.0.2'
   gem 'rspec-rails', '~> 3.8', '>= 3.8.2'
 end
 
+group :test do
+  gem 'database_cleaner', '~> 1.7'
```

Comment:
Do we still need to use this gem? @gonzaM

Reply:
Check here https://anti-pattern.com/transactional-fixtures-in-rails

Reply:
good one! I didn't know. Thanks 💪

Reply:
@t-romani we can try to remove the gem and its configuration and test if it works